GOVERNANCE.md

```diff
@@ -1 +1 @@
-See https://github.com/linkerd/linkerd2/blob/master/GOVERNANCE.md
+See https://github.com/linkerd/linkerd2/blob/main/GOVERNANCE.md
```

Comment:
(similarly, maybe this should also be a link?)

Reply:
hyperlinks are clickable without extra markdown!